spec/javascripts/components/intervention-spec.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-env jasmine, jquery */
+/* global GOVUK */
+
+describe('Intervention component', function () {
+  it('triggers a tracking event when the component is displayed', function () {
+    spyOn(GOVUK.analytics, 'trackEvent')
```

Comment:
This test doesn't test anything. You'll need to add a chunk of markup, initialise the component code on it and do and `expect` line to check that the spy has been called. Have a look at https://github.com/alphagov/govuk_publishing_components/blob/master/spec/javascripts/components/details-spec.js for an example.